app/cdk.json

```diff
@@ -1,3 +1,3 @@
 {
-  "app": "npx ts-node bin/medical-transcription-analysis.ts"
+  "app": "npx ts-node --skip-project bin/medical-transcription-analysis.ts"
```

Comment:
Why do we need the --skip-project flag here?

Reply:
`ts-node` looks for a tsconfig file, which we didn't have before so it was using default compiler options. We want to ignore the new tsconfig file for now while I think of a way to rework the project so all the TS pieces work well with one config or have separate configs.